.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -26,6 +26,6 @@ Self-reviews with GitHub's review UI are encouraged to point out the following:
 - Particularly important or core changes.
 - Items that may need further discussion.
 
-Thank you for contributing to PrairieLearn!
+Thank you for contributing to PrairieLearn! Etiquette for contributions can be found at <https://prairielearn.readthedocs.io/en/latest/contributing>.
```

Comment:
Nit, IMO angle brackets aren't necessary in comments.

```suggestion
Thank you for contributing to PrairieLearn! Etiquette for contributions can be found at https://prairielearn.readthedocs.io/en/latest/contributing.
```

Reply:
Other nit: I'd move the "Etiquette..." section to be its own comment on the first line, before even `# Description`, to maximize the odds that someone reads it. And to end on a nice happy positive "thanks!" note 🙂